tests/examples/no-matrix/dependencies.yaml

```diff
@@ -0,0 +1,48 @@
+files:
```

Comment:
Can we minimize the test cases in a future PR? The sample inputs are pretty big and it's hard to pick out what's essential for each test.

Reply:
Yes, we can definitely do that.